### DISCUSSION TITLE
Graphics glitch

### DISCUSSION THREAD

**Discussion by SVKPsycho** [OpenHantek/OpenHantek6022#226] [Q&A] [answered]
Hello community,
I am facing graphics glitch (see screenshot).
![openhantek6022_2](https://user-images.githubusercontent.com/69419792/134239888-4fd40315-2914-4039-91d9-aa826ae46d79.png)

 After launch, the app is stretched through two displays (both with res 1920x1080), same problem when using only one display. I cannot resize the toolbar, nor vertically resize whole window. I am using Garuda linux (arch based) with plasma 5.22.5 desktop.
Terminal verbose output shows no error (verbose level 2). Also tried --useGLES and --resetSettings.
[HantekOutput.txt](https://github.com/OpenHantek/OpenHantek6022/files/7205821/HantekOutput.txt)
Except of extremely wide toolbar, when I move some slider in scope it leaves marks at previous text location. 
The automatically generated config file is also included. I guess it's possible to set window geometry via config file but I don't understand meaning of values in ByteArray.
[DSO-6022BE_0D64B45EB12B.txt](https://github.com/OpenHantek/OpenHantek6022/files/7205907/DSO-6022BE_0D64B45EB12B.txt)

I tried installing from aur repo and also building from source, each time I removed whole config directory. Also tried installing OpenHantek with the same bug.
Can somebody please help me with this?

Thanks a lot!

**Comment by Ho-Ro** [comment, marked as answer]
> Also tried installing OpenHantek with the same bug.

You're talking about the old unmaintained version? So it seems to be a more general issue, maybe graphics HW/driver related.
What graphic HW and driver do you use?

Did you try --useGLSL150 option? But honestly speaking, I see no problem with OpenGL, it looks more like a Qt issue.

EDIT: See the solution below - switch from GTK3 system theme to "Fusion" style.

**Reply by SVKPsycho** [reply]
Hi, thanks for your quick reply.
I am using nvidia gpu in hybrid mode with intel integrated gpu and intel/nvidia latest drivers. Yesterday I gave it last chance starting it as sudo and it started with default white theme and everything worked as it was supposed to. So it's probably related with GTK3 theme, so maybe a QT problem? As user I use Breeze GTK3 theme.
*Edit: I am now using latest OH6022 version from github.

**Reply by Ho-Ro** [reply]
You could try the *Fusion* style, selectable in menu Oscilloscope/Settings/Colors:
![image](https://user-images.githubusercontent.com/12542359/134307988-50c246be-e3ef-41d4-80b7-616bdb557cd2.png)

**Reply by SVKPsycho** [reply]
Yeah, thank you very much sir. Fusion style solved my problem.

**Reply by Trelane92** [reply]
Had the same issue with multiple systems, using fusion style fixed! very difficult to use with the stretched screen otherwise.
